cypress/integration/login.js

```diff
@@ -21,7 +21,9 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Algimantas Vizgirda');
         cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.calendar--today').contains('24');

```

Comment:
Try to use built in JavaScript functions to get the current date whenever you run the test instead of hard-coding "24", since the test would not work tomorrow this way